nslsii/sync_experiment/sync_experiment.py

```diff
@@ -177,6 +182,14 @@ def main():
         help="Which beamline (e.g. CHX)",
         required=True,
     )
+    parser.add_argument(
+        "-e",
```

Comment:
Although `-p` Is already used, `-e` is non-intuitive. It's probably better to have no shortcut rather than a confusing one.
```suggestion
```